src/libMessage/Messenger.cpp

```diff
@@ -2552,6 +2615,23 @@ bool Messenger::SetLookupSetSeedPeers(vector<unsigned char>& dst,
                                     *result.add_candidateseeds());
   }
 
+  SerializableToProtobufByteArray(lookupKey.second, *result.mutable_pubkey());
+
+  Signature signature;
+  if (result.candidateseeds().size() > 0) {
+    // Serialize the candidate seeds to temp buffer, later use it to calculate
+    // the signature.
+    std::vector<unsigned char> tmp;
+    RepeatableToArray(result.candidateseeds(), tmp, 0);
+    if (!Schnorr::GetInstance().Sign(tmp, lookupKey.first, lookupKey.second,
+                                     signature)) {
+      LOG_GENERAL(WARNING, "Failed to sign transactions.");
```

Comment:
Error message is wrong. Please check the other parts of the code too for similar changes.